packages/kit/src/core/build/build_server.js

```diff
@@ -171,14 +171,14 @@ export async function build_server(
 
 	// prettier-ignore
 	fs.writeFileSync(
-		input.app,
-		template({
-			cwd,
-			config,
-			hooks: app_relative(hooks_file),
-			has_service_worker: service_worker_register && !!service_worker_entry_file
-		})
-	);
+    input.app,
```

Comment:
it looks like these may have been changed to spaces? but I'd have thought `pnpm format` or `pnpm lint` would catch it

Reply:
I took a look around the app, there seems to be spaces in stead of tabs everywhere :O 

Do you want to just ignore these for now and I can do a lint blitz in a seperate PR?

Reply:
Looks like this one was caused by an existing prettier-ignore, so fixed that